opencti-platform/opencti-graphql/src/modules/vocabulary/vocabulary.graphql

```diff
@@ -117,7 +117,7 @@ type Query {
     orderMode: OrderingMode
     filters: FilterGroup
     search: String
-  ): VocabularyConnection @auth(for: [KNOWLEDGE])
+  ): VocabularyConnection @auth(for: [KNOWLEDGE, INGESTION_SETINGESTIONS])
```

Comment:
Why in vocabularies ? (curious as well)

Reply:
To update RSS Feeds, you must to have access to vocabularies throw Openvocab query

Reply:
what usecase do we want to solve by adding this capability for vocabulary ? If we have a user with only access to INGESTION_SETINGESTIONS ? but then this user won't be able to see creators, authors, ... all these queries are protected under knolwedge capability.